prow/plugins.yaml

```diff
@@ -287,6 +287,15 @@ require_matching_label:
   prs: true
   regexp: ^(sig|wg|committee)/
   # Define needs-sig separately for issues so that we only comment on issues.
+- missing_label: needs-review
+  org: kubernetes
+  repo: kubernetes
+  issues: true
+  regexp: ^(triage)/
+  missing_comment: |
+    Awaiting SIG review/triage.
+    
+    If it's a valid issue, please assign the relevant tracking labels (kind,priority etc) and remove `needs-review`.
```

Comment:
I'm not sure this configuration does what you want based on the state machine diagram, but I'm also a bit confused by the terminology. My understanding is that this label is meant to be applied to PRs in the triage state in the diagram, but this will actually be applied to PRs in the waiting room as well. This could be avoided by changing the regexp to `^triage/|^needs-sig$`.

This message also suggests removing the `needs-review` label, but doesn't say how. Labels shouldn't be manually added or removed by humans, especially this label since the automation will just replace the label if a human tries to remove it.

Reply:
Thanks, changed the diagram, label and regex.

You're right about automatically removing the 'needs-sig-triage' label, and I wonder what the trigger would be. Perhaps by applying a 'lifecycle' label as in here: https://github.com/kubernetes/sig-release/issues/551 ? That should be decided for the bigger picture

Reply:
I don't mind if the label is applied in the waiting room, it's equally true.  That said, having been sig-triaged without a sig is a weird state to be left in.

Reply:
we're thinking of utilizing `lifecycle` labels as status indicators (lifecycle/ready to be worked on - lifecycle/active in progress - forgottenfrozen - stale etc), with exit criteria of `needs-triage` to be 
`regexp: ^(lifecycle)/(ready|active|frozen)$`